tests/keyword-detection.test.html

```diff
@@ -0,0 +1,38 @@
+<html>
+  <body>
+    <div data-ea-publisher="test"></div>
+    <main>
+      <p>
+        PyTorch is an important module for machine learning. PyTorch can use
+        your GPU to crunch numbers faster than a CPU.
+      </p>
+    </main>
+
+    <script type="module">
+      import { expect } from "@open-wc/testing";
+      import { runTests } from "@web/test-runner-mocha";
+      import { default as sinon } from "sinon";
+
+      import { wait, detectedKeywords, Placement } from "../index";
+
+      // Don't actually call the server
+      // https://sinonjs.org/releases/v17/stubs/
+      let stub = sinon.stub(Placement.prototype, "fetch");
+      const response_html =
+        "<div><!-- A real ad would be here normally --></div>";
+      const elem_placement = document.createElement("div");
+      elem_placement.innerHTML = response_html;
+      stub.resolves(elem_placement.firstChild);
```

Comment:
Realizing now that this is probably going to be in all the test cases, and could probably be a separate, shared import at some point.

Reply:
I'll see if it's easy to break it out into a shared lib.

Reply:
Also, with only a few test cases, it's not like we have too much redundancy in the tests yet. This can totally be saved for later, or if we ever need to update this block on all the test cases.